### PR TITLE
updates browser-tools orb

### PR DESCRIPTION
**Pull request by whereismyjetpack**

### CODE REVIEW

**Comment by whereismyjetpack** [comment]
@ajkiessl - I believe the failing spec to be a flaky one. I'm not sure if we should try to fix it, or xit.. what do you think?

**Comment by ajkiessl** [comment]
I pushed a change that should fix this and still test (I'm assuming) the same thing.  The `visually-hidden` class for that element was changed to `sr-only` at some point.  I think these accomplish the same thing.

File: spec/fixtures/vcr_cassettes/viewing_a_record/when_the_record_has_an_IIIF_manifest_URL/renders_the_Mirador_viewer_with_no_workspace_type.yml [hunk 1]
```diff
@@ -94,4 +94,106 @@ http_interactions:
       encoding: ASCII-8BIT
       string: var _GBSBookInfo = {};
   recorded_at: Thu, 25 May 2023 18:31:49 GMT
+- request:
+    method: get
```
Comment (whereismyjetpack) [inline spec/fixtures/vcr_cassettes/viewing_a_record/when_the_record_has_an_IIIF_manifest_URL/renders_the_Mirador_viewer_with_no_workspace_type.yml]: I wonder if we should do what we do In scholarsphere for this? 

https://github.com/psu-libraries/scholarsphere/blob/main/bin/ci-rspec#L54

I'm not sure we should be recording calls to get chromedriver, we also may want to allow chromedriver.storage.googleapis.com in our allowed hosts configuration?

Comment (whereismyjetpack) [reply spec/fixtures/vcr_cassettes/viewing_a_record/when_the_record_has_an_IIIF_manifest_URL/renders_the_Mirador_viewer_with_no_workspace_type.yml]: Oh i see you did revert this change.

**Review by ajkiessl — APPROVED** [review]

**Pull request merged**